source/includes/_endpoint-user.md

```diff
@@ -76,4 +106,64 @@ The new owner must also be part of the same account and should have
 }
 ````
 
-The new owner provided can be a user URL or a user email.
\ No newline at end of file
+The new owner provided can be a user URL or a user email.
+##### User Datasets
+
+`/account/users/{id}/datasets/`
+
+This URL is only accessible and available to account admins. 
+
+This Shoji catalog lists all the datasets that are owned by this user.
+
+
+##### User Visible datasets
+
+`/users/{id}/visible_datasets/`
```

Comment:
why not /datasets/visible/?

This is also odd because the behavior of account/users/id/datasets/ and /visible_datasets/ is different from the user's own datasets/ catalog.